test/source_test.go

```diff
@@ -320,12 +363,42 @@ index ea874f5..ba38f84 100644
 		}
 	}
 
-	t.Run("no patch", func(t *testing.T) {
-		t.Parallel()
-		testEnv.RunTest(baseCtx, t, func(ctx context.Context, gwc gwclient.Client) {
-			checkModule(ctx, gwc, "github.com/cpuguy83/tar2go@v0.3.1", baseSpec())
+	tests := map[string]struct {
```

Comment:
I'm thinking we probably don't need this extra test since it is going to add a fair amount of test time  and more potential for flakiness.

Reply:
ok, I drop this extra test